lib/pure/times.nim

```diff
@@ -1142,34 +1091,33 @@ proc parseToken(info: var TimeInfo; token, value: string; j: var int) =
     j += 4
   of "z":
     if value[j] == '+':
-      info.timezone = parseInt($value[j+1])
+      info.timezone = 0-parseInt($value[j+1]) * 3600
```

Comment:
Confusing precedence. * binds stronger than - but the spacing suggests otherwise. (Strong spaces would have been so cool...)